holoviews/tests/plotting/bokeh/test_group_label_tooltips.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
```

Comment:
I think we can move this test into bokeh/test_elementplot.py

Reply:
I added a new section in the existing class for hover tooltip tests. hope that's what you meant